### PR TITLE
Remove font logic from Shout since it is all dead code

### PR DESCRIPTION
**Pull request by jecisc**
Iâ¯deprecated some stuff that might be used by Spec

### CODE REVIEW

**Comment by MarcusDenker** [comment]
- [CI] test runs sometimes aborted due to SubscriptOutOfBounds error #12502

**Comment by jecisc** [comment]
I changed the base to be Pharo 12 and this will need to be updated to create a Deprecated12 instead of using Deprecated11

File: src/Deprecated11/SHRBTextStyler.extension.st [hunk 1]
```diff
@@ -0,0 +1,23 @@
+Extension { #name : #SHRBTextStyler }
+
+{ #category : #'*Deprecated11' }
```
Comment (privat) [inline src/Deprecated11/SHRBTextStyler.extension.st]: Deprecated12?

File: src/Deprecated11/SHRBTextStyler.extension.st [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+SHRBTextStyler >> font: aFont [
+
+	self deprecated: 'This variable is never used and was removed.'
```
Comment (privat) [inline src/Deprecated11/SHRBTextStyler.extension.st]: if never used, maybe just kill it?

Comment (jecisc) [reply src/Deprecated11/SHRBTextStyler.extension.st]: The variable was never used internally, but I don’t know if the class has a user that set the font. #font: is too common for me to check all senders and guess it the receiver is the styler or not ahah

File: src/Deprecated11/SHRBTextStyler.extension.st (hunk 1 above)
Comment (jecisc) [reply src/Deprecated11/SHRBTextStyler.extension.st]: Yeah I originally did this Pr on P11 and I need to update it. I’ll do it today

**Comment by MarcusDenker** [comment]
ReleaseTest>>testUndeclared

'''Found undeclared Variables:  textAttributesByPixelHeight in: MicSmalltalkTextStyler class>>#initialize'''

**Review by MarcusDenker — APPROVED** [review]

**Pull request merged**